rpc_jobs/standard_job_postmerge.yml

```diff
@@ -44,7 +44,7 @@
     concurrent: true
     FLAVOR: "performance1-1"
     IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
-    CRON: "@daily"
+    CRON: "{CRON_DAILY}"
```

Comment:
For some reason I couldn't respond to your comment. It would be nice to restrict the impact to Mondays only. Could two expressions be supplied, one for limiting the hours on Monday and the other allowing any time of the day?Although if it is a problem, having it still impact work on a Monday might still be a bad thing. Maybe it's easier to just see if this causes an issue and address it then

Reply:
Two expression can't be supplied within the macro, but each job could have:
```
 triggers:
   - timed: "{CRON_DAILY}"
   - timed: "{CRON_MONDAY}"
```

Or something like that. However I think that daily meaning the job runs 6/7 days will be fine for most jobs. Any that do have a problem can use the other pattern as [explained on the maintenance confluence page](https://rpc-openstack.atlassian.net/wiki/spaces/RE/pages/469794817/RE+Infrastructure+Maintenance+Window).